sql/tables.go

```diff
@@ -385,3 +395,8 @@ type TableNode interface {
 	Databaser
 	UnderlyingTable() Table
 }
+
+type MutableWrappedTableNode interface {
```

Comment:
noted above, is this redundant or intentional? if this is different than `MutableTableWrapper` maybe need a different verb than "rewrap"

Reply:
I'll work on names, they might be too general